rules/rule_manager.go

```diff
@@ -37,20 +44,22 @@ func (rm *ruleManager) getStaticRules(key string, value *string) map[staticRule]
 					qSat := sRule.qSatisfiable(key, value)
 					if qSat == qTrue || qSat == qMaybe {
 						out[sRule] = index
+						prioritized[sRule] = rm.rules[rule]
 					}
 				} else {
 					if sRule.satisfiable(key, value) {
 						out[sRule] = index
+						prioritized[sRule] = rm.rules[rule]
 					}
 				}
 			}
 		}
 	}
-	return out
+	return out, sortRulesByPriority(prioritized)
```

Comment:
Humm, I worry about the overhead of doing this for every call when the result is always the same. Every field value, which can be large, 20 million each crawl bootstrap. 
Maybe we need some debug logging or enable Go profiling to see how much of a hit this is going to be.